admin/assets/stylesheets/elements/_restarts.scss

```diff
@@ -59,11 +59,14 @@ input[type=radio]:checked ~ .hide {
   }
 }
 
-#didNotCompleteInfo {
-  padding-bottom: pem(5px);
+.restartReasonInfo {
```

Comment:
@GeorgeChatzigiannis Can we make it into kebab case (lower case and hyphen separated)? That's how we are naming new css classes unless they are ids. So it would be `restart-reason-info`

Reply:
@pris54 done